src/test/java/jpo/sdw/depositor/consumerdepositors/KafkaConsumerRestDepositorTest.java

```diff
@@ -74,6 +75,41 @@ public boolean loop(Invocation inv) {
       testKafkaConsumerRestDepositor.run("testTopic");
    }
 
+    @Test
+   public void runShouldDepositJSONMessage() {
+
+      List<ConsumerRecord<String, String>> crList = new ArrayList<ConsumerRecord<String, String>>();
+      crList.add(new ConsumerRecord<String, String>("key", 0, 0, "value", "{\"encodedMsg\":\"C4400000000680C0DE3\"}"));
```

Comment:
The method itself utilizes record.value for passing the JSON object, so I believe it's where we expect. Historically the existing test has been passing null, but it could provide more value as a generic string.